examples/gallery/histograms/scatter_and_histograms.py

```diff
@@ -52,13 +53,13 @@
 )
 
 # Shift the plot origin and add right margin histogram
-fig.shift_origin(yshift="-10.25c", xshift="10.25c")
+fig.shift_origin(yshift=f"-{fig_width + 0.25}c", xshift=f"{fig_width + 0.25}c")
 
 fig.histogram(
     horizontal=True,
-    projection="X2c/10c",
-    # Note that the y-axis annotation "Counts" is shown in x-axis direction
-    # due to the rotation caused by horizontal=True
+    projection=f"X2c/{fig_width}c",
+    # Note that the y-axis annotations, ticks, and label "Counts" are shown in x-axis
```

Comment:
```suggestion
    # Note that the y-axis ticks, tick labels, and axis label "Counts" are shown in x-axis
```

Reply:
For this histogram, the region is set to be `region=[ymin, ymax, 0, 0]`, so the vertical  axis is the actual X-axis and the horizontal axis is the actual "Y-axis". So, instead of saying:

> Note that the y-axis annotations, ticks, and label "Counts" are shown in x-axis direction due to the rotation caused by horizontal=True.

Maybe we should say something like:

> `horizontal=True` plots horizontal histograms, with X/Y axes flipped. I.e., the X axis is plotted vertically and the Y axis is plotted horizontally.

Reply:
We probably also have to update the histogram tutorial at https://github.com/GenericMappingTools/pygmt/blob/8b6f61da8d88d020e05ea12b015c630949df4b0c/examples/tutorials/advanced/cartesian_histograms.py#L82-L85

Reply:
Updated at cead753d95630b1bfccae39d4e21080b8690725d.